nifi-nar-bundles/nifi-poi-bundle/nifi-poi-processors/src/main/java/org/apache/nifi/processors/poi/ConvertExcelToCSVProcessor.java

```diff
@@ -101,6 +100,34 @@
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .build();
 
+    public static final PropertyDescriptor FIRST_ROW = new PropertyDescriptor
```

Comment:
If I understand the behavior correctly, this property should be named like 'Number of skip rows'. Also, we should mention that skips empty rows automatically regardless to this setting.